recipes/snowline/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "snowline" %}
+{% set version = "0.6.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 43d0455594b31808805bad60ac076dc8f91335a2e1ad6bfb94af0a7e4b002743
+
+build:
+  number: 0
+  # may need to later add "skip: True  # [win]" to disable Windows.
+  skip: true  # [py<30 or python_impl == "pypy" or win]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
+    - pip
+    - pypmc
+    - iminuit
+    - numpy
+    - pytest-runner
+
+  run:
+    - python
+    - numpy
+    - matplotlib-base
+    - corner
+    - pypmc
+    - iminuit
```

Comment:
These do not match the upstream metadata

https://github.com/JohannesBuchner/snowline/blob/6f9b5f767903ebf0537569ff85752a8533664b8a/setup.py#L23